packages/docs/sidebars.js

```diff
@@ -12,13 +12,21 @@ function titleCase(str) {
 function nestMarkdownFiles() {
   return markdownFiles()
     .sort((a, b) => a.split('.').length - b.split('.').length)
-    .reduce((acc, f) => {
+    .reduce((acc, f, i) => {
```

Comment:
Looks unused

Reply:
It is unused, I added that for some debugging purposes. I'll remove it.

Reply:
Reminder